game.py

```diff
@@ -117,3 +114,36 @@ def random_play_game(env,
     v = winner * default_v
     states = np.array(states)
     return states, v
+
+
+def play_against_random(env,
```

Comment:
Where is the model being used here?  Shouldn't it alternate between taking a random action and one from the state?

Reply:
whoops on it